GitUtilities.js

```diff
@@ -8,7 +8,18 @@ function hasTag(tag) {
     .then(() => true, () => false);
 }
 
-exports.init = dest => execa('git', ['init'], { cwd: dest, stdio: 'inherit' });
+exports.isGitRepo = dest =>
```

Comment:
what is this?

you could also just test for the existence of a `.git`

Reply:
i could but i figured this was a bit more robust if your in a nested folder